assert/assert.go

```diff
@@ -35,18 +37,32 @@ func DeepEqual(t *testing.T, variable string, actual, expected interface{}) bool
 		return true
 	}
 
+	t.Error("assert.DeepEqual failed for " + variable)
+
+	prettyDiffEnv := os.Getenv("GOBITS_PRETTY_DIFF")
+	if prettyDiffEnv == "" {
+		prettyDiffEnv = "false"
+	}
+	prettyDiff, err := strconv.ParseBool(prettyDiffEnv)
+	if err != nil {
+		t.Fatalf("Parsing GOBITS_PRETTY_DIFF failed: %s", err.Error())
+	}
```

Comment:
No need to die on parsing here, just do
```
prettyDiffEnv, _ := strconv.ParseBool(os.Getenv("GOBITS_PRETTY_DIFF"))
```
That will make it default to false on empty string or unknown values. (Same below.)

Reply:
butiful.